builtin/providers/google/resource_container_cluster.go

```diff
@@ -240,6 +289,28 @@ func resourceContainerClusterCreate(d *schema.ResourceData, meta interface{}) er
 		cluster.Network = v.(string)
 	}
 
+	if v, ok := d.GetOk("subnetwork"); ok {
+		cluster.Subnetwork = v.(string)
+	}
+
+	if v, ok := d.GetOk("addons_config"); ok {
+		addonsConfig := v.([]interface{})[0].(map[string]interface{})
+		cluster.AddonsConfig = &container.AddonsConfig{}
+
+		if v, ok := addonsConfig["http_load_balancing"]; ok {
+			addon := v.([]interface{})[0].(map[string]interface{})
+			cluster.AddonsConfig.HttpLoadBalancing = &container.HttpLoadBalancing{
+				Disabled: addon["disabled"].(bool),
```

Comment:
This will default to `false` if omitted, since you didn't specify this subattribute as required - is that intended behavior? Same thing below on line 310.